.github/compatibility-test-matrices/main/client-chain-a.json

```diff
@@ -5,6 +5,7 @@
   "test": [
     "TestClientUpdateProposal_Succeeds"
   ],
+  "relayer-type": ["COSMOS"],
```

Comment:
this change as it currently stands will not change the number of tests that run, once a second entry is added (`HERMES`) this will double the number of tests that run in each compatibility test matrix.

Reply:
should we call them `golang` and `rust` instead of `cosmos` and `hermes`? not super opinionated here but jw

Reply:
hm I agree golang could work, but I don't think `rust` would be a good name, the language doesn't really matter for a relayer I feel like the explicit name is more valuable information (as you could have many relayers written in rust or golang. )

Reply:
I think my vote would be for the binary name in lowercase. E.g. `hermes` or `rly`

Reply:
yeah I'm happy with `hermes/rly`